### ISSUE TITLE
Travis log is full

### ISSUE THREAD

**Issue by iSazonov** [PowerShell/PowerShell#2836]
Last time this error occurs during Travis CI tests:
```
1828The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).
```
Many records in the log is present:
```
Updating Help for module Microsoft.PowerShell.Core                                 Installing Help content...                                                      [oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo]                                                                                      
                                                                                 Updating Help for module Microsoft.PowerShell.Core                                 Installing Help content...                                                      [oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo]                                                                                      VERBOSE: Microsoft.PowerShell.Core: Updated 
/home/travis/build/PowerShell/PowerShell/src/powershell-unix/bin/Linux/netcorea
pp1.0/ubuntu.14.04-x64/publish/en-US/about_Session_Configurations.help.txt. 
Culture en-US Version 5.1.1.0                                                                                 Updating Help for module Microsoft.PowerShell.Core                                 Installing Help content...                                                      [oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo]                                                                                      
                                                                                 Updating Help for module Microsoft.PowerShell.Core                                 Installing Help content...                                                      [oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo]                                                                                      VERBOSE: Microsoft.PowerShell.Core: Updated 
/home/travis/build/PowerShell/PowerShell/src/powershell-unix/bin/Linux/netcorea
pp1.0/ubuntu.14.04-x64/publish/en-US/about_WMI_Cmdlets.help.txt. Culture en-US 

```

The proposal is to remove unnecessary progress bar by adding `$progressPreference = 'silentlyContinue' ` in Help tests.

/cc @vors @lzybkr

**Comment by daxian-dbw** [comment]
@Francisco-Gamino, could you please update the Updateable help tests to not write progress bar? Thanks!

**Comment by vors** [comment]
Reducing amount of output from particular test would not solve the root of the problem.
Should we just employee @JamesWTruher #2795 to all builds?

**Comment by iSazonov** [comment]
@vors There are two problems (really bugs). 1. The progress bar is not removed and remains trash. Seems #2795 is not addressed the trash. 2. Now progress bar still has performance problem. So it's better to disable the progress bar. It might even be a globally at Pester.

**Comment by vors** [comment]
Should we just completely disable progress reporting on unix, until it can be acceptable represented visually?

**Comment by iSazonov** [comment]
I think it will be OK in `Build.psm1` as _default_ parameter (`-Quite` ?) for CI tests. For "visually" we always can set `-Quite:$false`.

**Comment by SteveL-MSFT** [comment]
Do we still need to track this or is this resolved by https://github.com/PowerShell/PowerShell/pull/2795?

**Comment by iSazonov** [comment]
We have reduced the size of the Travis CI log by  #2795 and #2842.
Current size is 2.1 MB (max Travis CI log size is 4 MB). 0.5 MB from 2.1 MB is unneeded information about configure CI environment (progress bar of downloading packages, sources and etc.)

We will increase the number of tests for better code coverage and we can exhaust the limit.

You and I asked Pester team make output filter and show only failed tests in output. But as I understand we blocked because new Pester version is not ported.
